website/src/components/Changelog/Android.tsx

```diff
@@ -4,6 +4,38 @@ import Entry from "./Entry";
 export default function Android() {
   return (
     <Entries title="Android">
+      <Entry version="1.1.0" date={new Date("2024-06-27")}>
+        <ul className="list-disc space-y-2 pl-4 mb-4">
+          <li className="pl-2">
+            Introduces the new DNS routing system supported by 1.1.0 Gateways
+            which should result in much more stable connections for DNS
```

Comment:
```suggestion
            which results in much more stable connections for DNS
```

Let's not doubt ourselves :)

Reply:
Perhaps lets emphasis wildcard resources too!